vault/changelog.release.2022-06-21.md

```diff
@@ -0,0 +1,86 @@
+---
+id: 14h74cnp9no9joxavmo7gr6
+title: 0.100
+desc: ''
+updated: 1655744379419
+created: 1655737056633
+---
+
+Dendron 0.100 has sprouted  🌱
+
+**v100:** We will be releasing version 100 of Dendron next week! This will mark 100 releases and over 600 features shipped! We'll be launching on Product Hunt on Tuesday June 21st, 12:01AM PST. If Dendron has been of use to you, please leave feedback on our [product hunt page](https://www.producthunt.com/posts/dendron-v100) at that time - this would help us out a lot! 🙏
```

Comment:
Today marks 100 releases of Dendron!  Two years ago, we launched Dendron because we believed there was a [better way of managing knowledge](https://www.kevinslin.com/notes/e1455752-b052-4212-ac6e-cc054659f2bb.html). Over six hundre feature releases later, we think we're 5% of the way there in building such a system. 

To celebrate the v100 milestone, we're also launching on product hunt today! **If Dendron has been of use to you**, please leave **feedback on our [product hunt page](https://www.producthunt.com/posts/dendron-v100) today** (product hunt is heavily skewed for launch day feedback) - this would help us out a lot! 🙏